package.js

```diff
@@ -2,21 +2,21 @@ Package.describe({
   name: 'ostrio:autoform-files',
   summary: 'File upload for AutoForm using ostrio:files',
   description: 'File upload for AutoForm using ostrio:files',
-  version: '2.5.1',
+  version: '2.5.2',
   git: 'https://github.com/veliovgroup/meteor-autoform-file.git'
 });
 
 Package.onUse((api) => {
-  api.versionsFrom('METEOR@2.0');
+  api.versionsFrom(['2.0', '3.0-rc.0']);
 
   api.use([
     'check',
     'ecmascript',
     'mongo',
     'reactive-var',
     'templating@1.4.2',
-    'aldeed:autoform@7.0.0',
-    'ostrio:files@2.2.0'
+    'aldeed:autoform@8.0.0-rc.1',
+    'ostrio:files@3.0.0-beta.5'
```

Comment:
@harryadel @jankapunkt I think we shouldn't mainline versions, and add beta/rc as an option. Wdyt?

```js
[
    'aldeed:autoform@7.0.0 || 8.0.0-rc',
    'ostrio:files@2.2.0 || 3.0.0-beta.5',
]
```

Reply:
@dr-dimitru Please re-check